modules/customizing-the-login-page.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * web_console/customizing-the-web-console.adoc
+
+[id="customizing-the-login-page_{context}"]
+= Customizing the login page
+
+Create Terms of Service information with custom login pages. Custom login pages
+can also be helpful if you use a third-party login provider, like GitHub or
+Google, to show users a branded page that they trust and expect before being
+redirected to the authentication provider. You can also render custom error
+pages during the authentication process.
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+.Procedure
+
+. Run the following commands to create templates you can modify:
++
+----
+$ oc adm create-login-template > login.html
+$ oc adm create-provider-selection-template > providers.html
+----
+
+. Create the secrets:
++
+----
+$ oc create secret generic login-template --from-file=login.html -n openshift-config
+$ oc create secret generic providers-template --from-file=providers.html -n openshift-config
+----
+
+. Run:
++
+----
+$ oc edit oauths cluster
+----
+
+. Update the specification.
++
+----
+spec:
+  templates:
+    login:
+        name: login-template
+    providerSelection:
+        name: providers-template
+----
++
+Run `oc explain oauths.spec.templates` to understand the options:
```

Comment:
if the description changes, you'd have to change the docs, I would not post the output here

Reply:
> if the description changes, you'd have to change the docs, I would not post the output here

+1, sorry, I only meant to include the output in my comment as a reference. I agree with @stlaz here. I do think it makes sense to include the command itself.